test/src/main/scala/io/finch/test/package.scala

```diff
@@ -0,0 +1,12 @@
+package io.finch
+
+import com.twitter.util.Await
+
+package object test {
+  /**
+   * Syntax for testing an [[Endpoint.Mapped]].
+   */
+  implicit class MappedEndpointOps[In, A](endpoint: Endpoint.Mapped[In, A]) {
+    def lift(in: In): A = Await.result(endpoint.underlying(in)).value
```

Comment:
Do you think we should do both `liftToValue(in: In): A` and `liftToOutput(in: Int): Output[A]` so users can assert on content type, headers, status code, etc?